back/lib/participation_method/base.rb

```diff
@@ -114,8 +118,8 @@ def supports_inputs_without_author?
       true
     end
 
-    def supports_multiple_posts?
-      true
+    def allow_posting_again_after
```

Comment:
I was a little bit confused at first because I first assumed this would return a date (already taking into account the last post), but now I understand and it's clear. Just wondering if a different naming could have avoided my confusion, but not sure at all.